src/graphics/constants.js

```diff
@@ -1100,6 +1100,16 @@ export const UNIFORMTYPE_VEC2ARRAY = 21;
 export const UNIFORMTYPE_VEC3ARRAY = 22;
 export const UNIFORMTYPE_VEC4ARRAY = 23;
 
+// (bit-flags) shader stages for resource visibility on the GPU
+export const SHADER_STAGE_VERTEX = 1;
+export const SHADER_STAGE_FRAGMENT = 2;
+export const SHADER_STAGE_COMPUTE = 4;
```

Comment:
Notice the constants above are in the form `UNIFORMTYPE_<VALUE>` and not `UNIFORM_TYPE_<VALUE>`. So for consistency, I would go with `SHADERTYPE_<VALUE>`.

Reply:
we had a similar discussion not too long ago, and decided using _ to separate works is cleaner to read I think?

Reply:
The overwhelming majority are of the form `<TYPE>_<VALUE>`:

https://developer.playcanvas.com/api/pc.html

Some aren't, true. But do we really want to just casually mix the two instead of laying down some standards/rules here?

Reply:
I think we should discuss this next week?
This (existing one) is so hard to read `XRDEPTHSENSINGUSAGE_CPU`

Reply:
changing these to SHADERSTAGE_VERTEX ...